packages/merge-on-green/README.md

```diff
@@ -0,0 +1,21 @@
+# merge-on-green
```

Comment:
this is looking good, we should make sure to upstream these template changes to your generator.

Reply:
ah right, will do